backend/api/battles/urls.py

```diff
@@ -0,0 +1,18 @@
+from django.urls import path
+
+from api.battles.views import (
+    CreateBattleEndpoint,
+    CreateTeamEndpoint,
+    DetailBattleEndpoint,
+    ListBattlesEndpoint,
+)
+
+
+app_name = "battle"
+
+urlpatterns = [
+    path("battles/<status>/", ListBattlesEndpoint.as_view(), name="list_active"),
```

Comment:
Although this isn't wrong, I'd recommend using query params to filter the battles.
This line would become `path("battles/", ListBattlesEndpoint.as_view(), name="list_battles"),` and you'd deal with the query params in the `get_queryset` view's method.